lms/djangoapps/discussion_api/api.py

```diff
@@ -274,6 +277,7 @@ def get_thread_list(
         "last_activity_at".
     order_direction: The direction in which to sort the threads by. The only
         values are "asc" or "desc". The default is "desc".
+    discussion_id: Content-specific discussion topic ID
```

Comment:
@nasthagiri: Are you asking of an "inline discussion" can just be handled via its unique topic, assuming it has a unique topic? Is it possible to get some final help from @jimabramson before he's (your) gone? I will need to be able to answer these questions, but I can't yet with any confidence.